assets/js/themeLoader.js

```diff
@@ -1,16 +1,38 @@
 const body = document.body;
-const data = body.getAttribute("data-theme");
+const config = body.getAttribute("data-theme");
+
+const getThemeState = () => {
+    const lsState = localStorage.getItem("theme");
+    if (lsState) return lsState;
+
+    let state;
+    switch (config) {
```

Comment:
I feel, using switch/case is a nice idea for this particular use case (color theme change).